### PR TITLE
feat(www): adds quick introduction section

### PR DESCRIPTION
**Pull request by ixahmedxi**
## 🎯 Changes

image

Would like for someone to go over the content and maybe improve it to be more elaborative.

## ✅ Checklist

- [x] I have followed the steps listed in the [Contributing guide](https://github.com/trpc/trpc/blob/main/CONTRIBUTING.md).
- [x] If necessary, I have added documentation related to the changes made.
- [x] I have added or updated the tests related to the changes made.

### CODE REVIEW

File: www/src/pages/index.tsx [hunk 1]
```diff
@@ -118,6 +119,8 @@ function Home() {
           <Features />
         </section>
 
+        <QuickIntro />
```
Comment (KATT) [inline www/src/pages/index.tsx:122]:
```suggestion
        {isV10 && <QuickIntro />}
```

File: www/src/components/QuickIntro.tsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+                  ))}
+                </div>
+              ))}
+            </pre>
+          )}
+        </Highlight>
+      </div>
+      <div className="flex-1">
+        <div className="flex flex-col justify-center gap-3 lg:flex-row lg:items-center lg:justify-start">
+          <div className="grid w-6 h-6 rounded-full bg-zinc-500 place-items-center">
```
Comment (jlalmes) [inline www/src/components/QuickIntro.tsx:88]: NIT: `bg-zinc-200 dark:bg-zinc-500`

File: www/src/components/QuickIntro.tsx [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+  const client = createTRPCProxyClient<AppRouter>({
+    url: "http://localhost:3000"
+  });
+
+  const res = await client.greeting.query({ name: "John" });
+
+  console.log(res.text);
+}`;
+
+type StepProps = {
```
Comment (jlalmes) [inline www/src/components/QuickIntro.tsx:44]: NIT: do these need to be `readonly` properties?

File: www/package.json [hunk 4]
```diff
@@ -32,6 +32,7 @@
     "clsx": "^1.1.1",
     "cssnano": "^5.1.13",
     "docusaurus-preset-shiki-twoslash": "^1.1.38",
+    "prism-react-renderer": "^1.3.5",
```
Comment (jlalmes) [inline www/package.json:35]: are we sure this doesn't negatively affect page load times? would be great to use `shiki` if possible - no need to maintain two different syntax highlighter dependencies, themes etc.

**Comment by jlalmes** [comment]
Heads up, looks like this PR breaks the `<TwitterWall />` layout on desktop... I'm not sure quite how though.

**Review by sachinraja — APPROVED** [review]

**Pull request merged**